lib/service/auth_service.dart

```diff
@@ -168,6 +168,7 @@ class AuthService {
       if (e.code != "requires-recent-login") rethrow;
       throw AuthenticationRequiredException();
```

Comment:
Should we redirect user to login flow again here?

Reply:
Currently it shows a dialog asking the user whether they want to sign out or not.